sippy-ng/src/component_readiness/CompReadyUtils.js

```diff
@@ -282,8 +282,8 @@ export function getUpdatedUrlParts(
     confidence: confidence,
     pity: pity,
     minFail: minFail,
-    ignoreDisruption: ignoreDisruption,
-    ignoreMissing: ignoreMissing,
+    ignore_disruption: ignoreDisruption,
+    ignore_missing: ignoreMissing,
```

Comment:
Why the mismatch between camel and snake case?

Reply:
the left side is used for the api call; the right side is the value I'm using.

Reply:
the left side is used for the api call; the right side is the value I'm using.

Reply:
Why? The rest of the values here are camel case above these two lines